### DISCUSSION TITLE
Project name/branding

### DISCUSSION THREAD

**Discussion by javulticat** [NewPipeX/NewPipeX#14] [Ideas]
Please use this discussion space to share and discuss any ideas/thoughts/suggestions you have regarding the name and/or branding (logo, colors, themes, other design aspects, etc.) this project should use.

That could range from keeping everything more-or-less identical to how `polymorphicshade` had things, to a full redesign.

If someone shares an idea you like, make sure to give it a positive reaction (↑, 👍, ❤️, 🚀, 🎉, etc.), as ideas that obtain the most support among the community will be included in the official poll (TBA) on renaming/rebranding.

**Comment by javulticat** [comment]
I'll start.  When originally creating this repo, I explored a few rebranding ideas.  I ended up designing a very rough (I'm _not_ a designer) draft of a new logo for one name idea (which is a play on how many **x**'s there are in: "New Pipe **x** SponsorBlock **x** Return YouTube Dislikes"):

### NewPipeX
![image](https://github.com/javulticat/NewPipe/assets/31746850/7fad9f28-d3be-4558-8b41-1ce1623e96e0)

**Reply by SyCoREAPER** [reply]
Logo looks good but resembles the Bluetooth logo at a glance. Tried some AI but can't get it to do what I want. 

Are you looking to keep it NewPipe-ish or complete rebrand?

As for names, some ideas:

NuPipe or EnhancedPipe or PipeTube or NewTube - SponsorBlock x Return YouTube Dislike

**Reply by javulticat** [reply]
I guess something to consider is that I'm **not** planning on making any significant changes to the application functionality that has been inherited from `polymorphicshade`'s fork, which changed its app name to `NewPipe SponsorBlock` but did no noticeable rebranding to differentiate itself from upstream `NewPipe` beyond that, yet still ended up being very successful, was added to a major F-Droid repo, etc.

So I guess, in a way, the extent to which this project should be renamed/rebranded rather depends on if one thinks that `polymorphicshade`'s fork ought to have done more extensive renaming/rebranding to distance itself more from upstream `NewPipe` (and, if so, what the benefit of doing so would have been).  IMO, it was probably for the best that `polymorphicshade`'s fork did not do too much to differentiate itself from `NewPipe`, especially since it's essentially indistinguishable from `NewPipe` other than having a few additional options in the settings.

And, since this project intends to remain the same as its parent fork in that regard, I'm not quite sure what a complete rebrand away from the `NewPipe` name would accomplish (other than possibly some confusion).  That's why, if the app's name were going to be changed, I tend to prefer using a name that keeps its `NewPipe` lineage obvious (e.g., `NewPipeX`).  I mean, the way I originally discovered the existence of `NewPipe SponsorBlock` was that I read about `NewPipe` in some article, searched for "`NewPipe`" in F-Droid, then saw `NewPipe SponsorBlock` appear in the search results.  Dropping `NewPipe` from the name entirely could remove a significant way that people are able to discover this app.

Also, keep in mind that I am _not_ a designer, so any kind of rebranding that would require doing design work more complex than the very basic stuff I'm capable of (e.g., changing icon colors: Chromium ![Chromium](https://github.com/javulticat/NewPipe/assets/31746850/7916628b-86f3-4397-877b-c44082dd6e59) vs. Chrome ![Chrome](https://github.com/javulticat/NewPipe/assets/31746850/ee323b6a-d799-4f44-ae96-f125ae3a5636) vs. Vanadium ![Vanadium](https://github.com/javulticat/NewPipe/assets/31746850/5643ccfd-b9ed-43a8-a858-13312a213d24)) probably doesn't have a very realistic chance of actually getting done, unless we have designers in our community who are willing to help out.

**Reply by SyCoREAPER** [reply]
That's fair but as someone else pointed out, this is now your fork. I wouldn't think it wrong or narcissistic to name your Fork. 
It was polys choice to keep the name to the parent no matter how big or little the change, I don't think it should matter or affect what you decide to do. 

I would at least recommend you consider it after the "big update" official NewPipe will be pushing if you plan to support it then and thereafter. The update is supposedly a supposedly a significant overhaul. As such, the fork would almost be it's own thing. 

But it's entirely up to you how you go about it. I just think it's worth some credit.

**Comment by delit** [comment]
I'm a graphic designer with 20 years of experience. If you know the name of the project or the direction your heading, I could help with the icon (correct grid and vector design etc).

**Comment by extremelyonline** [comment]
I like the NewPipeX name. Personally, I much prefer the idea to just change the icon color instead of redesigning the icon. Aesthetically, a simple icon is much more pleasing.

**Reply by SyCoREAPER** [reply]
X is visually appealing as a title but IMO an overdone naming convention to denominate a difference or to standout.
